package/thingino-webui/files/var/www/x/p/header.cgi

```diff
@@ -48,6 +48,7 @@ Pragma: no-cache
 <li class="nav-item dropdown">
 <a aria-expanded="false" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" href="#" id="dropdownSettings" role="button">Settings</a>
 <ul aria-labelledby="dropdownSettings" class="dropdown-menu">
+<li><a class="dropdown-item" href="config-audio.cgi">Audio</a></li>
```

Comment:
you don't this one after recent refactoring of webui. all files wish names starting from `config-` will be included to the menu automatically.

Reply:
renamed to just `audio.cgi`